CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Changed
+- **BREAKING**: Fix `normalize` for empty strings ([#315](https://github.com/MetaMask/eth-sig-util/pull/315))
```

Comment:
Note that this affects zero as well, in the same way. We should mention that as well